corehq/motech/repeaters/templates/repeaters/partials/bootstrap3/attempt_history.html

```diff
@@ -30,7 +30,7 @@
                             {% endif %}
                             {% if attempt.message %}
                                 <div class="well record-attempt" style="font-family: monospace;">
-                                    {{ attempt.message }}
+                                    {{ attempt.message|escape|linebreaksbr }}
```

Comment:
I've escaped these since they can contain data from the HTTP response which may not be safe.